spark/src/main/scala/io/substrait/spark/expression/FunctionMappings.scala

```diff
@@ -67,6 +67,7 @@ class FunctionMappings {
     s[Count]("count"),
     s[Min]("min"),
     s[Max]("max"),
+    s[First]("any_value"),
```

Comment:
While you can treat `any_value` with `first`, I don't think you can treat `first` as `any_value`. Is this mapping bi-directional?

Reply:
Sure, my first proposal was to add the `first` function to `functions_aggregate_general.yaml`, but this suggested was given as an alternative.
https://github.com/substrait-io/substrait/pull/697#issuecomment-2315684133

Reply:
Ah, if Spark itself handles `First` as `any_value` this makes sense. Though truth be told if I'm being pedantic this sounds like Spark doesn't actually support `First` correctly. Not a Substrait problem though.